test/parallel/test-fs-assert-encoding-error.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+require('../common');
+const assert = require('assert');
+const fs = require('fs');
+
+const options = 'test';
+const unknownEncodingMessage =
+  new RegExp(`^Error: Unknown encoding: ${options}$`);
+
+assert.throws(() => {
+  fs.readFile('path', options, () => {});
```

Comment:
Can you create a `noop` variable, give it the value of `() => {}`, then just reuse that.